src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -210,11 +203,15 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
       }
     });
   }
+  // find user that submitted the change request
+  //const person = await prisma.user.findUnique({ where: { userId: updated.submitterId } });
+  const person = body.submitter;
```

Comment:
the commented out line is more correct, you need to get the person from prisma using the submitterId, then include the usersettings in that object